src/EditorFeatures/Test2/ReferenceHighlighting/VisualBasicReferenceHighlightingTests.vb

```diff
@@ -272,5 +272,27 @@ End Class
 
             VerifyHighlights(input)
         End Sub
+
+        <WorkItem(1904, "https://github.com/dotnet/roslyn/issues/1904")>
+        <WorkItem(2079, "https://github.com/dotnet/roslyn/issues/2079")>
```

Comment:
If this test doesn't prevent 2079 from regressing then don't add that WorkItem attribute.  Similarly for the comment below.